sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/GenericDockerVolumeProvider.java

```diff
@@ -27,6 +28,9 @@ public GenericDockerVolumeProvider(
       volumeNameUnescaped = providedVolumeName.get();
     } else {
       volumeNameUnescaped = providedVolumeName.filter(name -> !name.isEmpty()).orElse(serviceName) + '_' + podType;
+      if (containerPath.isPresent()) {
+        volumeNameUnescaped = volumeNameUnescaped + '-' + containerPath.get();
```

Comment:
I think we should use underscore (`_`) here instead of hyphen (`-`). Otherwise it will not work for NetApp.
```suggestion
        volumeNameUnescaped = volumeNameUnescaped + '_' + containerPath.get();
```